workshop/content/docs/setup/index.md

```diff
@@ -144,15 +141,11 @@ Below we also see that the configuration is explictly set to `pygeoapi-config.ym
 <div class="termy">
 
 ```console
-$ curl -O https://github.com/geopython/pygeoapi/blob/master/docker/default.config.yml
-$ vi default.config.yml
 $ docker run -p 5000:80 \
```

Comment:
Don't know if this will work. The pygeoapi config from the project repo only refers to test data.
So somehow it should use data from `/pygeoapi/mydata`.

Reply:
i guess we should be more explicit here, not all understands how to work with vim, better to use their favourite text editor to edit the file and then mount it into the container